packages/sites/ortho-site/webapp/wdkCustomization/js/client/records/Sequences.tsx

```diff
@@ -662,23 +663,7 @@ export function RecordTable_Sequences(
         } as CSSProperties
       }
     >
-      {warningText && (
-        <div
-          style={{
-            display: 'flex',
-            flexWrap: 'wrap',
-            borderLeft: '.2em solid rgb(225, 133, 133)',
-            borderRight: '.2em solid rgb(225, 133, 133)',
-            padding: '.5em 1em',
-            background: 'rgb(255, 228, 228)',
-            gap: '1em',
-            marginBottom: '1em',
-            fontWeight: 500,
-          }}
-        >
-          {warningText}
-        </div>
-      )}
+      {warningText && <NoteBox type="error">{warningText}</NoteBox>}
```

Comment:
we should either change `warningText` to `errorText` or use the `type='warning'`. Having "warning" text in an error Notebox is confusing.

Reply:
This is one of those things where someone wanted the box to be red, even though it isn't strictly an error. I will change it to `errorText` for clarity.

Reply:
Done!